packages/taquito-rpc/src/types.ts

```diff
@@ -1720,6 +1721,89 @@ export interface ConstantsResponseCommon {
 
 export type Ratio = { numerator: number; denominator: number };
 
+export interface ConstantsResponseProto020
+  extends Omit<
+    ConstantsResponseProto019,
+    | 'adaptive_rewards_params'
+    | 'blocks_per_stake_snapshot'
+    | 'dal_parametric'
+    | 'issuance_weights'
+    | 'liquidity_baking_subsidy_weight'
+    | 'preserved_cycles'
+    | 'smart_rollup_reveal_activation_level'
+  > {
+  adaptive_issuenace_force_activation: boolean;
+  adaptive_rewards_params: {
+    center_dz: {
+      denominator: string;
+      numerator: string;
+    };
+    growth_rate: {
```

Comment:
growth_rate duplicated in ConstantsResponseProto019

Reply:
it's rewritten because the parent object `adaptive_rewards_params` is omitted due to a subset of its children being updated. We need to rewrite all the sub properties